src/Command.php

```diff
@@ -31,7 +31,7 @@ class Command implements CommandInterface
      * @param mixed  $handler
      * @param mixed  $options
      */
-    public function __construct($name, $handler, $options = null)
+    final public function __construct($name, $handler, $options = null)
```

Comment:
Is that one the warnings from phpstan?

I don't want to restrict the constructor. If someone wants to overwrite the constructors - I'm fine. So why to declare it final? If you still can overwrite the constructor (even more: it means the final keyword has no effect on constructors) I would especially not add this keyword.

In conclusion I would not add final for one or the other reason:
a) they should not be final
b) it might even be ignored from php

Reply:
PHPStan emits:

```
Unsafe usage of new static().                                                      
See: https://phpstan.org/blog/solving-phpstan-error-unsafe-usage-of-new-static
```

But fair enough, I'll remove those

Reply:
ah, I see! Thats an intresting point. Maybe we should then use `self` instead. but please in a different PR if you want to solve it.

anyway it sounds odd to change the constructor and not the implementation of the static create method as it is just a syntax sugar to avoid the parenthesis around the new keyword: `Option::create('h')->setDescription('Show help')` instead of `(new Option('h'))->setDescription('Show help')`